### PR TITLE
UI: Added Cards for Course Detail Page

### LINKED ISSUE #5: Create Course Card components for Course Page
Part 3 of #2 

The component screenshot:
image

Please see the parent issue for more details and full design

### PR DESCRIPTION
**Pull request by rizwan2000rm**
**Issue Number**

fixes #5

**Describe the changes you've made**

Added a Card Component for Course Detail Page, also added a wrapper container for horizontal scrolling list.

**Checklist**

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by rizwan2000rm** [comment]
@Abhishek-kumar09 @KeenWarrior Please review this PR.

File: src/components/cardContainer/cardContainer.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+// import Container from "@material-ui/core/Container";
```
Comment (Abhishek-kumar09) [inline src/components/cardContainer/cardContainer.js:3]: Delete this and remove extra newline

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
No new package is added thus there should be no new changes in package-lock.json.

Please copy and paste the the lock file form main repo to revert these changes

File: src/components/cardContainer/cardContainer.js (hunk 1 above)
Comment (rizwan2000rm) [reply src/components/cardContainer/cardContainer.js]: Done

**Comment by rizwan2000rm** [comment]
@Abhishek-kumar09 I have copied the exact whole package-lock file from the repo's latest commit it still shows that it has been edited but has the exact same content now.

**Review by KeenWarrior — APPROVED** [review]

**Pull request merged**